### ISSUE TITLE
Active DBTabItem is not announced by VoiceOver on iOS

### ISSUE THREAD

**Issue by d-koppenhagen** [db-ux-design-system/core-web#5287]
### Which generators are affected?

- [ ] All
- [ ] HTML
- [x] React
- [ ] Angular
- [ ] Vue
- [ ] Web components
- [ ] Power Apps

### Reproduction case

1. Open the [tabs Demo page](https://design-system.deutschebahn.com/core-web/version/v4.0.1/components/data-display/tabs/overview?page=examples) on iOS / Safari
2. Start Voice Over
3. Navigate through the tabs

### Expected Behaviour

The currently active tab should be announced as active.

### Browser version

Safari

### Add any other context about the problem here.

The announcement works fine with VoiceOver on macOS (Chrome, Safari, MS Edge).

probably it is somehow related to:
- https://github.com/db-ux-design-system/core-web/issues/5046
- https://github.com/db-ux-design-system/core-web/issues/4247

### Which DB business unit do you work for?

DB Systel GmbH (please name your customer below)

### What project are you working on?

Mobility4All

**Comment by d-koppenhagen** [comment]
I found this page here which seems to use a similar pattern / implementation:

https://www.digitala11y.com/aria-selected-state/

The matrix shows vor iOS Voice Over for "MUST convey true value by indicating that the element is currently selected" a state of "none".

It seems that iOS voice over does not support this kind of announcement?

**Comment by mfranzke** [comment]
@d-koppenhagen I actually get the announcement "Auswahl, Tab 2, tab". But I've identified another bug https://github.com/db-ux-design-system/core-web/issues/5288, could this be related to your experience ?
